### PR TITLE
Use joho/godotenv to parse .env style secrets

### PR DESCRIPTION
**Pull request by 7er**

### CODE REVIEW

File: pkg/synchronizer/synchronizer_test.go [hunk 1]
```diff
@@ -74,6 +74,16 @@ func TestToSecretDataWithEnv(t *testing.T) {
 	}, payload)
 }
 
+func TestToEnvMultilineEncoded(t *testing.T) {
+	envPayload       = []byte("FOO=\"one line\\ntwo line\\n\"\n")
```
Comment (tronghn) [inline pkg/synchronizer/synchronizer_test.go:78]: I'd like to see the addition of a second environment variable (e.g. `BAR=BAZ`) in the test payload as well.

**Review by tronghn — COMMENTED** [review]
Please run `make test` locally to ensure that the formatter is run.

File: pkg/synchronizer/synchronizer.go [hunk 2]
```diff
@@ -169,25 +168,8 @@ func ToSecretData(msg google.PubSubMessage, payload map[string]string) kubernete
...
-		key := keyval[0]
-		val := keyval[1]
-		if _, ok := env[key]; ok {
-			return nil, fmt.Errorf("duplicate environment variable on line %d", n+1)
-		}
-		env[key] = val
-	}
-	return env, nil
+	env, err := godotenv.Unmarshal(data);
+	return env, err
```
[the remark this answers is not in the document]
Comment (7er) [reply pkg/synchronizer/synchronizer.go:172]: Good idea

File: pkg/synchronizer/synchronizer_test.go (hunk 1 above)
Comment (7er) [reply pkg/synchronizer/synchronizer_test.go:78]: Will do

**Comment by 7er** [comment]
@tronghn Thanks for handholding, this was my very first line of golang code.

**Review by tronghn — APPROVED** [review]
LGTM. Thank you for contributing @7er!

**Pull request merged**